flink-python/pyflink/datastream/connectors/file_system.py

```diff
@@ -599,8 +620,26 @@ def with_output_file_config(self, output_file_config: OutputFileConfig) \
                 output_file_config._j_output_file_config)
             return self
 
+        def _with_row_data_converter(self, row_type: 'RowType') -> 'FileSink.BulkFormatBuilder':
```

Comment:
What about renaming it to something else as the given parameter isn't a converter?